Santouka/js/main.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+{
+  // All, [elem, elem, elem, elem]
```

Comment:
I need some comment 
1. why they are comment out
2. why they stay here ( fixed in the future? )

Reply:
I deleted it!